src/main/java/frc/robot/Constants.java

```diff
@@ -13,7 +20,106 @@
+
+    public static final double DRIVING_P = 0.04;
+    public static final double DRIVING_I = 0;
+    public static final double DRIVING_D = 0;
+    public static final double DRIVING_FF = 1 / DRIVE_WHEEL_FREE_SPEED_RPS;
+    public static final double DRIVING_MIN_OUTPUT = -1;
+    public static final double DRIVING_MAX_OUTPUT = 1;
+
+    public static final double TURNING_P = 1;
+    public static final double TURNING_I = 0;
+    public static final double TURNING_D = 0;
+    public static final double TURNING_FF = 0;
+    public static final double TURNING_MIN_OUTPUT = -1;
+    public static final double TURNING_MAX_OUTPUT = 1;
+
+    public static final IdleMode DRIVING_MOTOR_IDLE_MODE = IdleMode.kBrake;
+    public static final IdleMode TURNING_MOTOR_IDLE_MODE = IdleMode.kBrake;
+
+    public static final int DRIVING_MOTOR_CURRENT_LIMIT_AMPS = 50; // amps
+    public static final int TURNING_MOTOR_CURRENT_LIMIT_AMPS = 20; // amps
+  }
+
+  public static final class NeoMotorConstants {
+    public static final double FREE_SPEED_RPM = 5676;
+  }
+
+  public static final class DriveConstants {
+    /**Driving Parameters - Note that these are not the maximum capable speeds of
+    the robot, rather the allowed maximum speeds*/
+    public static final double MAX_SPEED_METERS_PER_SECOND = 4.8;
+    public static final double MAX_ANGULAR_SPEED_RAD_PER_SECONDS = 2 * Math.PI; // radians per second
+
+    /**Chassis configuration*/
+    public static final double TRACK_WIDTH = Units.inchesToMeters(26.5);
+    /**Distance between centers of right and left wheels on robot*/
+    public static final double WHEEL_BASE = Units.inchesToMeters(26.5);
+    /**Distance between front and back wheels on robot*/
+    public static final SwerveDriveKinematics DRIVE_KINEMATICS = new SwerveDriveKinematics(
+        new Translation2d(WHEEL_BASE / 2, TRACK_WIDTH / 2),
+        new Translation2d(WHEEL_BASE / 2, -TRACK_WIDTH / 2),
+        new Translation2d(-WHEEL_BASE / 2, TRACK_WIDTH / 2),
+        new Translation2d(-WHEEL_BASE / 2, -TRACK_WIDTH / 2));
+
+    /**Angular offsets of the modules relative to the chassis in radians*/
+    public static final double FRONT_LEFT_CHASSIS_ANGULAR_OFFSET = -Math.PI / 2;
+    public static final double FRONT_RIGHT_CHASSIS_ANGULAR_OFFSET = 0;
+    public static final double BACK_LEFT_CHASSIS_ANGULAR_OFFSET = Math.PI;
+    public static final double BACK_RIGHT_CHASSIS_ANGULAR_OFFSET = Math.PI / 2;
+
+    /** SPARK MAX CAN IDs */
+    public static final int FRONT_LEFT_DRIVING_CAN_ID = 11;
+    public static final int REAR_LEFT_DRIVING_CAN_ID = 13;
+    public static final int FRONT_RIGHT_DRIVING_CAN_ID = 15;
+    public static final int REAR_RIGHT_DRIVING_CAN_ID = 17;
+
+    public static final int FRONT_LEFT_TURNING_CAN_ID = 10;
+    public static final int REAR_LEFT_TURNING_CAN_ID = 12;
+    public static final int FRONT_RIGHT_TURNING_CAN_ID = 14;
+    public static final int REAR_RIGHT_TURNING_CAN_ID = 16;
```

Comment:
These need to be connected with commas for the comment to apply to all, e.g.
```
    public static final int FRONT_LEFT_DRIVING_CAN_ID = 11, REAR_LEFT_DRIVING_CAN_ID = 13;
```

Reply:
Same comment applies in some other spots so take a look through. Note this only works when they're all the same type (`int` in this case)